docs/docs/examples/core-controllers/jtc-example.md

```diff
@@ -184,6 +184,8 @@ This message defines 2 of the variables that JTC can use in order to execute a t
 1. `frames`: a list of the Cartesian frames that you would like to traverse, in the order that they should be visited.
 2. `times_from_start`: a list of times (in seconds) measured from the start, indicating when JTC should reach each
 frame.
+3. `durations`: a list of absolute durations (in seconds) that correspond to each waypoint (mutually exclusive with 
```

Comment:
This is wrong though in this context, I would probably change this example to use durations instead because its easier to understand